Add a new configurable option to the windows installer for static mode (#5958)
grafana/agent: docs/sources/static/set-up/install/install-agent-on-windows.md

1. Double-click on `grafana-agent-installer.exe` to install Grafana Agent.

   Grafana Agent is installed into the default directory `C:\Program Files\Grafana Agent`.
   
   The following options are available:
   

>    - The [windows_exporter integration][windows_exporter_config] can be enabled with all default windows_exporter options. 

Author: This was hardcoded to /latest/ but should be corrected now

   If you are using Powershell, make sure you use triple quotes `"""http://example.com"""` around the URL parameter.

## Silent install with `-config.expand_env`

> It is possible to enable [-config.expand-env][flags] during a silent install.

Reviewer: ```suggestion
You can enable [-config.expand-env][flags] during a silent install.
```
